### DISCUSSION TITLE
Cannot export as REFI-QDA

### DISCUSSION THREAD

**Discussion by peterikeler** [ccbogel/QualCoder#820] [Q&A]
Hi,

I'm currently using QualCoder 3.3 for a large interview analysis project - with much work already completed - and need to export to REFI-QDA format. But when I attempt to do this (Project --> Export --> REFI-QDA export) it simply does not run any export or generate any new file. Other export options do work, such as Export codebook as ODT file, but not the key REFI-QDA one. I have also tried this in version 3.2 with the same error. Any guidance or help greatly appreciated!

Thank you,
Peter

**Comment by ccbogel** [comment]
HI. It must be crashing somewhere along the way when creating the export. Can you use the manual install and let me know what the command line/terminal says?
Alternatively, if you feel comfortable with this, can you send me the zip of the project (wither my ccbogel@hotmail.com or my work email, which you should be able to find) and I can test to see what the problem is.

**Reply by lmselby** [reply]
I am using QualCoder 3.6. I was having a problem exporting my project as REFI-QDA after I initially imported an REFI-QDA project from another software program into QualCoder. The original .qdpx from Atlas.ti had a copy saved into the hidden .qualcoder file in my C drive. I saved the project the D drive and was trying to export the QualCoder project as .qdpx to a folder in the D drive when I received this message:

  **File "qualcoder\__main__.py", line 1896, in refi_project_export

  File "qualcoder\refi.py", line 1907, in __init__

  File "qualcoder\refi.py", line 1995, in export_project

FileExistsError: [WinError 183] Cannot create a file when that file already exists: 'C:\\Users\\Lynn\\.qualcoder\\Deportation-rch-proj_KMP_20190405_LMS_20250624.zip' -> 'C:\\Users\\Lynn\\.qualcoder\\Deportation-rch-proj_KMP_20190405_LMS_20250624.qdpx'**

When I deleted the .qdpx file and the zip file corresponding to the project in the C hidden folder, I was then able to export the QualCoder project as .qdpx for MAXQDA. This .qdpx for MAXDQA worked in MAXQDA 24, but I deleted the original QualCoder project file by messing with the hidden folder in the C drive so I could not easily go back to working with the QualCoder projct.  This is a confusing situation for me, my default project directory for QualCoder has always been in the D drive. I can restore the QualCoder project from trash back to the .qualcoder folder in the C drive, but it loses some of the previous coding work that had been done and I am looking for another way to export a QualCoder project as .qdpx without this workaround. Please share any comments you have about how the hidden .qualcoder folder in the C drive works. Thanks in advance.

**Reply by ccbogel** [reply]
Hello,
I tested importing an exporting (within the C drive on Windows). And I did not get this issue, normally these extra files are deleted by qualcoder.
However, yes, you can delete those specific files within the hidden .qualcoder folder. That folder is used to store settings and some other temporary files are created in there when qualcoder is running. 

When exporting  - I am wondering if it might also have a problem recognising the D drive, this is just a guess for now. Perhaps, as a suggestion, export it to say the Desktop, then copy that file across to your D drive.

For exporting a qdpx, this is what QualCoder does:
Inside the .qualcoder folder it creates a temporary folder, then zips it to a .zip then renames it to a .qdpx , Then it copies this .qdpx to the final chosen folder - which should not be the .qualcoder folder.
It does not check and delete these files within the .qualcoder folder beforehand, so I have added code to 3.7 to fix that. However, you have done this yourself, which fixed the problem.

Regards this comment:
" I can restore the QualCoder project from trash back to the .qualcoder folder in the C drive"

**Do not store your QualCoder projects inside the .qualcoder folder**. This folder is _only_ meant to be used for QualCoder itself to store some information and, make other temporary files as it needs, when it is running.

Instead choose another location, such as the Documents folder.

I hope this helps, regards Colin
